turbo_alignment/trainers/dpo.py

```diff
@@ -293,6 +293,10 @@ def compute_loss(
         ratio = -F.logsigmoid(log_odds)
         losses = self.beta * ratio
 
+        nll_loss = -policy_chosen_logps
+
+        losses += nll_loss
```

Comment:
субъективно так приятнее
```suggestion
        losses = -policy_chosen_logps + self.beta * ratio
```